Sources/Backup/LocalBackup/OASettingsHelper.mm

```diff
@@ -104,10 +106,26 @@ - (instancetype)init
     if (self)
     {
         _currentBackupVersion = kVersion;
+        _exportTasks = [NSMutableDictionary dictionary];
     }
     return self;
 }
 
+- (OAImportAsyncTask *)getImportTask
+{
+    return _importTask;
+}
+
+- (void)setImportTask:(OAImportAsyncTask *)importTask
+{
+    _importTask = importTask;
+}
+
+- (void)removeExportTaskForFilepath:(NSString *)filePath
+{
+    [_exportTasks removeObjectForKey:filePath];
```

Comment:
Potentially unsafe object removal in a multithreaded environment. Use the @synchronized (self) {} macro